Tests/UpstreamIntegrationTests/UpstreamIntegrationTests.swift

```diff
@@ -188,11 +757,157 @@ class UpstreamIntegrationTests: XCTestCase {
             MobileCore.configureWith(appId: "94f571f308d5/6b1be84da76a/launch-023a1b64f561-development")
         }
     }
-    
+
     // TODO: create specific listeners for type: Edge + source: * (wildcard) and capture all the response handles for the test event
     // TODO: create specific listeners for error responses
     func registerEdgeLocationHintListener(_ listener: @escaping EventListener) {
         MobileCore.registerEventListener(type: FunctionalTestConst.EventType.EDGE, source: "locationHint:result", listener: listener)
     }
-}
 
+    // MARK: - Instrumented Extension helpers
```

Comment:
L767->L892 Since these are copied from the functional test helpers, I would like to extract those so they can be reused across both integration&functional tests. 
Feel free to include that into another PR, but let's at least remove them from this PR if possible or add a todo to remove the duplicated code